google/cloud/bigquery/_pandas_helpers.py

```diff
@@ -240,7 +163,7 @@ def bq_to_arrow_data_type(field):
     if field_type_upper in schema._STRUCT_TYPES:
```

Comment:
FYI: I'm hoping to use this helper in https://github.com/googleapis/python-bigquery-dataframes/pull/85 but might make sense for me to duplicate this code if we don't want dependencies on private APIs.

Reply:
I don't have plan to move this function, but indeed it feels more reliable to duplicate, mostly because it's private.